bedrock/privacy/templates/privacy/index.html

```diff
@@ -33,120 +33,57 @@
 {% endblock %}
 {% block footnote %}
   <section id="contact" class="section-content">
-      <h4>{{ _('Contact Mozilla') }}</h4>
+    <h4>{{ ftl('privacy-index-contact-mozilla') }}</h4>
+    <p>{{ ftl('privacy-index-if-you-want-to-make-a-correction') }}</p>
 
-      <p>
-        {%- trans -%}
-          If you want to make a correction to your information, or you have any questions about our
-          privacy policies, please get in touch with:
-        {%- endtrans -%}
-      </p>
-
-      <p lang="en" dir="ltr" itemscope itemtype="http://schema.org/Organization">
-        <span itemprop="name">Mozilla Corporation</span><br>
-        <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
-          {{ _('Attn:') }} <span itemprop="name">Legal Notices - Privacy</span><br>
-          <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
-          <span itemprop="addressLocality">Mountain View</span>,
-          <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span><br>
-          <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
-        </span>
-      </p>
+    <p lang="en" dir="ltr" itemscope itemtype="http://schema.org/Organization">
+      <span itemprop="name">Mozilla Corporation</span><br>
+      <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+        {{ _('Attn:') }} <span itemprop="name">Mozilla - Privacy</span><br>
```

Comment:
Nit: unwrap old-format string if it's hard-coded?
```suggestion
        Attn: <span itemprop="name">Mozilla - Privacy</span><br>
```